Wrappers/Python/cil/processors/RingRemover.py

```diff
@@ -26,30 +26,29 @@ class RingRemover(Processor):
     
     '''
         RingRemover Processor: Removes vertical stripes from a DataContainer(ImageData/AcquisitionData) 
-        the algorithm in https://doi.org/10.1364/OE.17.008567
+        using the algorithm in https://doi.org/10.1364/OE.17.008567
 
-    '''
-        
-    def __init__(self, decNum, wname, sigma, info = True):
-        
-        '''
-    
         Parameters
         ----------
-        decNum : number of wavelet decompositions
+        decNum : int
+            Number of wavelet decompositions
         
-        wname : (str) name of wavelet filter from pywt 
-            Example: 'db1' -- 'db35', 'haar'
+        wname : str
+            Name of wavelet filter from pywt e.g. 'db1' -- 'db35', 'haar'
     
-        sigma : Damping parameter in Fourier space.
+        sigma : float 
+            Damping parameter in Fourier space
```

Comment:
Is there any guidance you can give the user in a couple of sentences on how to interpret and use these? E.g. a larger sigma will lead to larger damping in Fourier space and potentially a more powerful ring remover but with the side effect of... Similarly, trying different number of decompositions of types of filter will have different effects...

Are there sensible defaults for these or is it very data dependent?

Reply:
Thanks Margaret - good idea! I've added some guidance and defaults